### PR TITLE
Update Koruri

### PR DESCRIPTION
**Pull request by unasuke**

### CODE REVIEW

File: Casks/font-koruri.rb [hunk 1]
```diff
@@ -1,9 +1,8 @@
 cask :v1 => 'font-koruri' do
-  version '20141224'
-  sha256 '2de09d84d05955018dd2b053ec912a94edb6320579bacfb96347d5fe8eb13872'
+  version '20150701'
+  sha256 '8a5ae87ffccc6c6a30312e82a9f3bdd7f1cdc6f84ab5113f45c32573a3a1d307'
 
-  # sourceforge.jp is the official download host per the vendor homepage
```
Comment (vitorgalvao) [inline Casks/font-koruri.rb]: If the host changes but is still not the same as the homepage, this comment need to be updated, not removed.

Needs a `# github.com is the official download host per the vendor homepage` comment ([example](https://github.com/caskroom/homebrew-cask/blob/cb487ad405f74cb1f26b93e3f3e8412c82a22e34/Casks/0ad.rb#L5)).

File: Casks/font-koruri.rb [hunk 2]
```diff
@@ -1,9 +1,8 @@
...
-  url "http://dl.sourceforge.jp/koruri/62469/Koruri-#{version}.tar.xz"
+  url "https://github.com/Koruri/Koruri/releases/download/Koruri-#{version}/Koruri-#{version}.tar.xz"
```
Comment (vitorgalvao) [inline Casks/font-koruri.rb]: Could you add a `appcast 'https://github.com/Koruri/Koruri/releases.atom'` line under this one?

File: Casks/font-koruri.rb (hunk 1 above)
Comment (unasuke) [inline Casks/font-koruri.rb]: OK. I will add that comment.

**Comment by unasuke** [comment]
Thank you to check my commit and comment.
Fix commit will be late...(few hours maybe)

**Comment by ndr-qef** [comment]
Thank you.

**Pull request merged**